### PR TITLE
First draft at karate's CI

### PR DESCRIPTION
**Pull request by GabrielFleischer**
First draft at karate's CI

For now, it will only support backends

### CODE REVIEW

**Review by jbsv — DISMISSED** [review]
LGTM, it really does :-D

**Review by Warhedgehog — DISMISSED** [review]
LGTM

File: .github/workflows/karate.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
+name: Integration Tests
+
+on:
+  schedule:
+    # Run the ci every day at 19:00
+    - cron:  '00 19 * * *'
```
Comment (pierluca) [inline .github/workflows/karate.yml:6]: how long does it take ? 
By default I'd strongly suggest running it in the early morning, say 4 am, as it's typical for students to work in the evening and 7pm may run "while" they're merging stuff.

File: .github/workflows/karate.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: "zulu"
+          java-version: "11"
+
+      - name: Setup repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of Sonar analysis
```
Comment (pierluca) [inline .github/workflows/karate.yml:23]: we're not running Sonar, though, right? The comment might be a bit outdated 😁

**Review by pierluca — APPROVED** [review]
Minor remarks, but LGTM ! 👍

File: .github/workflows/karate.yml [hunk 3]
```diff
@@ -0,0 +1,114 @@
+name: Integration Tests
+
+on:
+  schedule:
```
Comment (pierluca) [inline .github/workflows/karate.yml:4]: also, is this running on the main branch by default ?

File: .github/workflows/karate.yml (hunk 2 above)
Comment (GabrielFleischer) [reply .github/workflows/karate.yml:23]: This might be an artifact of a copy-paste 😅

File: .github/workflows/karate.yml (hunk 3 above)
Comment (GabrielFleischer) [reply .github/workflows/karate.yml:4]: I have no idea

File: .github/workflows/karate.yml (hunk 1 above)
Comment (GabrielFleischer) [reply .github/workflows/karate.yml:6]: It takes around 15 minutes.

I can definitely move it to 4 am.

Comment (GabrielFleischer) [reply .github/workflows/karate.yml:6]: It seems to be the default behavior, yes :

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

File: .github/workflows/karate.yml (hunk 3 above)
Comment (GabrielFleischer) [reply .github/workflows/karate.yml:4]: It seems to be the default behavior, yes :

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

**Review by pierluca — APPROVED** [review]

**Pull request merged**